templates/tonic-pyml-deployment.yaml

```diff
@@ -85,14 +85,9 @@ spec:
         ports:
         - containerPort: 7700
         resources:
-          limits:
-            memory: "8Gi"
-            {{- if ((.Values.resources).pyml_gpu).amount }}
-            nvidia.com/gpu: {{ .Values.resources.pyml_gpu.amount }}

```

Comment:
This moving into a different name may actually need some revised documentation in the `values.yaml` and/or sample.

Reply:
We unfortunately cannot break the chart in this fashion; however, I think we can devise a transitionary model for it.

```
{{- $resources := coalesce (((.Values).resources).pyml_service) (dict "limits" (dict) "requests" (dict "memory" "512Mi" "ephemeral-storage" "1Gi")) -}}
{{- $legacyGpuAmount := (((.Values).resources).pyml_gpu).amount -}}
{{- $newGpuAmount := (get $resources.limits "nvida.com/gpu") }}
{{- if and (not $newGpuAmount) $legacyGpuAmount -}}
{{- $_ := (set $resources.limits "nvidia.com/gpu" $legacyGpuAmount) -}}
{{- end -}}
```

And then templating with `$resources`

```yaml
resources:
    {{- toYaml $resources | nindent 10 }}}
```